From: Roland Rosenfeld <roland@debian.org>
Date: Sat, 23 Sep 2006 23:37:03 +0200
Last-Update: 2025-06-22
Subject: Change the global default from standard.Cautious to standard.Medium,
 which is similar to the old 3.0.3 behavior and doesn't change too much.
Forwarded: not-needed

--- a/match-all.action
+++ b/match-all.action
@@ -10,8 +10,16 @@
 +client-header-tagger{css-requests} \
 +client-header-tagger{image-requests} \
 +client-header-tagger{range-requests} \
++deanimate-gifs{last} \
++filter{refresh-tags} \
++filter{img-reorder} \
++filter{banners-by-size} \
++filter{webbugs} \
++filter{jumping-windows} \
 +hide-from-header{block} \
++hide-referrer{conditional-block} \
 +server-header-tagger{content-type} \
++session-cookies-only \
 +set-image-blocker{pattern} \
 }
 / # Match all URLs
